### ISSUE TITLE
option to NOT open in a new tab doesn't work with firefox containers

### ISSUE THREAD

**Issue by isingasimplesong** [fastaddons/GroupSpeedDial#244]
When unticking the option to open in new tabs, dials should open in same tab as GSD is. It does not work if the dial has to open into a specific firefox container

**Steps To Reproduce**

- use firefox
- set up any website to automatically open into a specific container
- untick GSD option to open dial in a new container
- --> regular dials open in same tab when containerized dials open in a new tab

**Expected behavior**

every dial should open in same tab, container or no container

**Device info:**
 - operating system: Linux
 - Browser and version: firefox 111.0.1
 - GroupSpeedDial version: 22.3

**Comment by isingasimplesong** [comment]
yes, i did assigned specific pages to open in a specific container, and that part work as intented. Sorry if i was not clear : the issue is that those pages always open in a NEW tab, when i want them to open in the CURRENT tab (GSD tab), like any other pages.

currently, when clicking on a dial, if it's assigned to a container, it opens in a new tab, when not assigned to a container, it opens in place

**Comment by Juraj-Masiar** [comment]
I'm trying it now and it's pretty interesting.
If I open new tab and click on a dial that points to an assigned page, it will open it in new tab.
But if I open two new tabs and do the same, it will correctly close one of the tabs.

In any case, this is not handled by my addon, this is the "Firefox Multi-Account Containers" addon issue, so it's best to report it to them.

Also, note that existing tab cannot be "converted" into container tab. So when the page suppose to open in a container, new tab must be created - and current tab should be closed. And this is somehow buggy when the tab is being opened from the addon page.
